templates/sonar_demo.html

```diff
@@ -43,16 +45,18 @@
     // get FFT results and render it on the spectrogram
     fft.event_listeners.push((fft_magnitudes) => {
       if (fft_magnitudes != null) {
-        app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+        //app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+          app.push(fft_magnitudes, {z_scale: 500000});
```

Comment:
It's okay to comment things out when trying different parameters, but when you're happy with how things are working, can you take out the commented code? In this case, can we merge the old line and new line so we end up with `app.push(fft_magnitudes.slice(395, 415), {z_scale: 500000});`.